src/Assets/Standard Assets/Bugsnag/BugsnagBehaviour.cs

```diff
@@ -50,10 +50,13 @@ void Awake()
     /// <param name="hasFocus"></param>
     void OnApplicationFocus(bool hasFocus)
     {
-      if (hasFocus && Bugsnag.Client.Configuration.AutoCaptureSessions)
-      {
-        Bugsnag.Client.SessionTracking.StartSession();
-      }
+      Bugsnag.Client.SetApplicationState(hasFocus);
+    }
+
+    void OnApplicationPause(bool paused)
+    {
+      var hasFocus = !paused;
+      Bugsnag.Client.SetApplicationState(hasFocus);
```

Comment:
@fractalwrench what do you think of this code now, is the intent clearer?

Reply:
lgtm!